server/services/generateReportService.ts

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+//@ts-ignore
+import { Legacy } from 'kibana';
```

Comment:
If we are going to move away from the Legacy kibana framework, will this import pose a problem after we move away from that?

Reply:
This imports the Elasticsearch Plugin type from Legacy for communicating with ES plugin. Cannot find a solution to not use legacy for now, also referred to other OD plugins. Any suggestions?

Reply:
No worries if also referred to in other OD plugins- was just worried about potentially accruing additional tech debt.